query-engine/connector-test-kit/src/test/scala/writes/scalarLists/CreateMutationListSpec.scala

```diff
@@ -90,7 +90,8 @@ class CreateMutationListSpec extends FlatSpec with Matchers with ApiSpecBase {
          |  }){optStrings, optInts, optFloats, optBooleans, optEnums, optDateTimes }
          |}""",
       project = project,
-      errorCode = 1
+      errorCode = 0, // 1
+      errorContains = """Error in query graph construction: InputError(\"The `set` argument was not provided for field `optStrings` on `ScalarModel`"""
```

Comment:
I think this should be an instance of the user facing MissingRequiredValue error, wdyt?

Reply:
sounds right :+1